api/services/ocean/spark/paths/clusterApplications.yaml

```diff
@@ -0,0 +1,79 @@
+summary: Ocean Spark
+
+get:
+  summary: "List Applications in Cluster"
+  description: >
+    List the Spark applications in a specific cluster.
+    This endpoint is paginated (cursor pagination).
+  operationId: "OceanSparkClusterApplicationList"
+  tags:
+    - "Ocean Spark"
+  parameters:
+    - $ref: "../../../../commons/parameters/nonRequiredAccountId.yaml"
+    - $ref: "../parameters/path/oceanSparkClusterId.yaml"
+    - $ref: "../parameters/query/oceanSparkJobId.yaml"
+    - $ref: "../parameters/query/oceanSparkApplicationSubmissionSource.yaml"
+    - $ref: "../parameters/query/oceanSparkApplicationState.yaml"
+    - $ref: "../parameters/query/userId.yaml"
+    - $ref: "../parameters/query/searchTerm.yaml"
+    - $ref: "../parameters/query/minCreatedAt.yaml"
+    - $ref: "../parameters/query/maxCreatedAt.yaml"
+    - $ref: "../parameters/query/minUpdatedAt.yaml"
+    - $ref: "../parameters/query/maxUpdatedAt.yaml"
+    - $ref: "../parameters/query/pageSize.yaml"
+    - $ref: "../parameters/query/after.yaml"
+    - $ref: "../parameters/query/before.yaml"
+    - $ref: "../parameters/query/order.yaml"
+    - $ref: "../parameters/query/orderBy.yaml"
```

Comment:
@arnarpall it looks those params are accepted by the optimizer but not yet used in the `bigdata-spark-service`.
Note that this is the `/ocean/spark/cluster/.../app/` route, not the cluster-agnostic route.

Should we remove them from the doc?

Reply:
No this is correct. They are all used in the `bigdata-spark-service`.

Reply:
Thanks! Are you going to add them to the optimizer in an upcoming PR @arnarpall ?

Reply:
Sorry, my answer does not make sense: I'm getting confused between comments. Thanks!